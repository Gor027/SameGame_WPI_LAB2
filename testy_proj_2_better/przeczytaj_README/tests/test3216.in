jwjw  wwj w jwwjjw jwwwjjwjw  jjw jj w  jw j  jjjjjwwww wwjwwjw 
wj  jw wjjwjjwjwww j   j j  wj jjw  j wj wj  wwj jjw
w w wjjw jwj wjw jwjjjjw jj j j  w ww  w jw 
