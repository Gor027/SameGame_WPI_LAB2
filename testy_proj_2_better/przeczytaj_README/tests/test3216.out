       w  w  wwj w j   j w               w
  j jwwwjjwjjwjwjw jwwwj jj w jjww jj  j wj  jj jjw
jjwwwjjwjjwjjwjwwjwjjjjwjjjwjjjjjwjwwwwwjjwjwwjjjjwwwwwwjwwjw
